### ISSUE TITLE
Add support to highlight single lines of code

### ISSUE THREAD

**Issue by bdemers** [okta/okta-developer-docs#2993]
It would be great if we could bold/highlight single (or a range of lines) of code in our snippets in order to draw attention a specific bit of code in a block:
https://vuepress.vuejs.org/guide/markdown.html#line-highlighting-in-code-blocks

This _looks_ like it should be possible today, but I wasn't able to get it to work locally.
I'm guessing it's related to our use of [markdown-it-attrs](https://www.npmjs.com/package/markdown-it-attrs):
 https://github.com/okta/okta-developer-docs/blob/master/packages/%40okta/vuepress-site/.vuepress/config.js#L286

I tried a few things without any success:
* Updating the version of `markdown-it`
* Adding a separate plugin: [markdown-it-highlight-lines](https://github.com/egoist/markdown-it-highlight-lines)

> **NOTE:** the Vuepress 2.x (beta) also mentioned they forked the `highlight-lines` plugin, so if we are planning on upgrading sometime after 2.x is released, then it's possible this could _just work_.

TL;DR: Highlighting, a single line (or lines) of markdown _should_ work via something like:
````md
```js{2-3}
console.log("dull code")
console.log("cool code")
console.log("more cool code")
console.log("other dull code")
```
````

**Comment by janethu-okta** [comment]
Hi Brian,
Create a JIRA ticket for the team to look into your suggestion.
https://oktainc.atlassian.net/browse/OKTA-485123

**Comment by annejuan-okta** [comment]
Hi Brian,

Just to follow up, we've implemented your request with this PR: https://github.com/okta/okta-developer-docs/pull/4929

Thanks!
